src/components/Checkbox/Checkbox.module.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../assets/variables.scss";
+
+.checkbox {
+  -webkit-appearance: none;
```

Comment:
Instead of adding prefixes by hand here and there, I'd add `appearance` only and use tool like https://github.com/postcss/autoprefixer to add other prefixes during build